template.yaml

```diff
@@ -2373,6 +2373,41 @@ Resources:
       EventInvokeConfig:
         MaximumRetryAttempts: 0
 
+  EventForwardingRule:
+    Type: AWS::Events::Rule
+    Properties:
+      Name: ForwardFromInternalToNvaEventBusRule
+      EventBusName: !Ref InternalBus
+      EventPattern:
+        detail:
+          responsePayload:
+            topic:
+              - "PublicationService.Resource.Deleted"
```

Comment:
I actually checked the value against the java constant…do I get a prize? 🏆 You do, because they match!